test/capture.test.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Component < Phlex::HTML
+	def view_template
+		yield if block_given?
+	end
+end
+
+UNSET = Object.new.freeze
+
+test "capturing nothing returns an empty string" do
+	output = UNSET
+	view_context = ActionController::Base.new.view_context
+
+	Component.render_in(view_context) do |c|
+		output = c.capture {} # rubocop:disable Lint/EmptyBlock
```

Comment:
I think Rubocop won’t complain if there’s a comment in the block explaining why it’s empty.